src/test/fixture/.vscode/settings.json

```diff
@@ -18,5 +18,6 @@
     "other": true,
     "comments": false,
     "strings": true
-  }
+  },
+  "security.workspace.trust.enabled": false
```

Comment:
This is an unrelated change, right?

Reply:
Vscode 1.57 introduced a trusted authors confirmation modal, that interrupts tests, so this line disables this check during tests running.